fe/fe-core/src/main/java/org/apache/doris/journal/bdbje/BDBEnvironment.java

```diff
@@ -339,13 +339,12 @@ public void removeDatabase(String dbName) {
         
         if (names != null) {
             for (String name : names) {
-                // We don't count epochDB
-                if (name.equals("epochDB")) {
-                    continue;
+                try {
+                    long db = Long.parseLong(name);
+                    ret.add(db);
+                } catch (NumberFormatException e) {
+                    // non-journal db, such as "epochDB", will throw this exception. No need to deal with it.
```

Comment:
better add a debug log so that we can see which database is skipped.

Reply:
done